kolibri/plugins/coach/assets/src/views/common/assignments/AssignmentDetailsModal.vue

```diff
@@ -217,6 +256,7 @@
         formIsSubmitted: false,
         showServerError: false,
         showTitleError: false,
+        instantReportVisibility: true,
```

Comment:
Should this be dynamic rather than hard-set? For example, when I open an exam for editing, should this be set to the value previously saved to the exam?

Reply:
Oi, just realized that you've set it in mounted - curious if you tried assigning it to `instantReportVisibility: this.assignment.instant_report_visibility || true;` here rather than in mounted below?

Reply:
I've just read that the Nullish coalescing operator (??) is mainly supported on newer browsers so I've decided to set it as `instantReportVisibility: this.assignment.instant_report_visibility !== false`, because `false !== false` evaluates to false, and for all other values, it is set to true.

Reply:
Update: I went with setting `instantReportVisibility: this.assignment.instant_report_visibility`, since that is what previously happened in the mounted hook anyway. 

Setting it as: `instantReportVisibility: this.assignment.instant_report_visibility !== false`, caused lessons to also contain a `instant_report_visibility` which broke the tests, and only quizzes should have this field. 

This field is set to true by default in `kolibri/plugins/coach/assets/src/composables/quizCreationSpecs.js`, so when a new quiz is created, it has the correct setting. So it was unnecessary to set it in mounted(), thanks for pointing this out!